composer.json

```diff
@@ -51,6 +51,11 @@
       "dev-master": "0.2-dev"
     }
   },
+  "scripts": {
+    "tests": [
+      "php ./vendor/phpunit/phpunit/phpunit -v"
```

Comment:
i think that this is not needed. you can do just `vendor/bin/phpunt` (is short and works the same)